util/dashboard_config.py

```diff
@@ -190,11 +190,11 @@ def prepare_ontologies(ontologies, ontology_dir, dashboard_dir, make_parameters,
             continue
 
         if download:
-            logging.info(f"Downloading {o}...")
+            logging.info("Downloading %s...", o)
             try:
-                urllib.request.urlretrieve(ourl, ont_path)
+                download_file(ourl, ont_path)
             except Exception:
-                logging.exception(f'Failed to download {o} from {ourl}')
+                logging.exception("Failed to download %s from %s", o, ourl)
```

Comment:
I personally prefer f strings? You dont?

Reply:
It's not recommended for logging messages. https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
Very interesting!!! Thanks, learned something new here!